app/views/projects/_form.html.slim

```diff
@@ -0,0 +1,36 @@
+= form_for Project.new, {remote:true}  do |f|
```

Comment:
shouldn't be calling `Project.new` in a view... better to do `@project = Project.new` in the controller and pass `@project` to the view to use.